src/popover/popover.directive.ts

```diff
@@ -33,6 +41,10 @@ export class PopoverDirective implements OnInit, OnDestroy {
    * Currently only supports "body".
    */
   @Input() public container: string;
+  /**
+   * Specifies if popover should be closed after clicking outside
+   */
+  @Input() public popoverCloseOnClickOutside: boolean;
```

Comment:
@valorkin, and what with setting this behavior via config? In the event-solution we would need to have both event and property to keep the functionality.